tao_compiler/mlir/disc/transforms/lhlo_legalize_roots_to_loops.cc

```diff
@@ -922,31 +930,44 @@ Value createSharedMemory(OpBuilder& b, Location loc, int64_t size,
   return alloc.getResult();
 }
 
+LogicalResult emitInThreadRowReduction(
+    OpBuilder& b, Location loc, Operation* op,
+    mlir::Block::args_iterator acc_iter, ValueRange& input_index,
+    SmallVector<Value, 4>& yield_values_for_j) {
+  if (!isRank2RowReduction(op)) {
```

Comment:
The function is just a "in thread reduction", not necessarily to be named as "in thread row reduction". 
And this check is not necessary as well.

Reply:
Renamed, with slight modification inside the function.